packages/core/src/tokens/index.ts

```diff
@@ -0,0 +1,35 @@
+import breakpoints from '../js/breakpoints'
+import layers from '../js/layers'
+import layout from '../js/layout'
+import type from '../js/type'
+import motion from '../js/motion'
+import * as colors from '../js/colors'
+
+const capitalizeFirstLetter = (str: string) => {
+  return str.charAt(0).toUpperCase() + str.slice(1)
+}
+
+type Tokens = {
+  breakpoints: typeof breakpoints
+  layers: typeof layers
+  layout: typeof layout
+  type: typeof type
+  motion: typeof motion
+} & typeof colors
+
+export const tokens = Object.entries({
+  breakpoints,
+  layers,
+  layout,
+  type,
+  motion,
+  ...colors
+}).reduce((acc, [group, obj]) => {
+  const keys: Record<string, string> = {}
+  Object.keys(obj).forEach(key => {
+    keys[key] = `var(--ps${capitalizeFirstLetter(group)}${capitalizeFirstLetter(
```

Comment:
To me, design tokens are usually abstracted from a tech implementation such as css custom props. Design tokens can be tricky enough to talk about and have mutual understanding about, that I am wary of stripping that distinction here.

Reply:
This is temporary call it what you want. It's a utility for migration alone.